src/autoyast-rnc/bootloader.rnc

```diff
@@ -86,7 +86,7 @@ boot_root = element boot_root { "true" | "false" }
 boot_boot = element boot_boot { "true" | "false" }
 boot_extended = element boot_extended { "true" | "false" }
 boot_mbr = element boot_mbr { "true" | "false" }
-smt = element smt { "true" | "false" }
+cpu_mitigations = element cpu_mitigations { "nosmt", "auto", "off", "manual" }
```

Comment:
Looks like the same problem as in https://github.com/yast/yast-installation-control/pull/81/files#diff-1e14931051b4c04ff21df76fc3588959R232

Reply:
yep, you are right.